src/config/datasets.yml

```diff
@@ -57,6 +57,22 @@ collections:
       - title: Storage Documentation
         src: alos-dem-storage.html
 
+  alos-fnf-mosaic:
+    category: Land use/Land cover
+    headerImg: ./images/alos-fnf-hero.png
+    tabs:
+      - title: Example Notebook
+        src: alos-palsar-example.html
+        launch: datasets/alos-palsar/alos-palsar-example.ipynb
+
+  alos-palsar-mosaic:
+    category: SAR
```

Comment:
FYI @mmcfarland, adding this new "category". Hope that's OK (I assume these map to the headers on the explorer dataset dropdown).

Reply:
Yep, that's right. They don't need to be defined elsewhere, should work as expected.